src/components/repository-list-item.component.js

```diff
@@ -87,7 +88,7 @@ const renderTitle = (repository, showFullName) =>
 
       <Icon
         containerStyle={styles.extraInfoIcon}
-        name="repo-forked"
+        name="ellipsis"
```

Comment:
@jouderianjr  Hi, can you explain why these 2 icons are modified?

Reply:
My bad, I was testing the vector-icon issue and I added in the commit by mistake. Nice catch! 🙈